includes/wc-update-functions.php

```diff
@@ -2175,3 +2175,70 @@ function wc_update_440_insert_attribute_terms_for_variable_products() {
 function wc_update_440_db_version() {
 	WC_Install::update_db_version( '4.4.0' );
 }
+
+/**
+ * Update DB version to 4.5.0.
+ */
+function wc_update_450_db_version() {
+	WC_Install::update_db_version( '4.5.0' );
+}
+
+/**
+ * Sanitize all coupons code.
+ *
+ * @return bool True to run again, false if completed.
+ */
+function wc_update_450_sanitize_coupons_code() {
+	global $wpdb;
+
+	$coupon_id      = 0;
+	$last_coupon_id = get_option( 'woocommerce_update_350_last_coupon_id', '0' );
```

Comment:
i think this option name was missed during rename, should be `woocommerce_update_450_last_coupon_id`

Reply:
Oops, sorry I missed that!